api/apps/api/src/modules/projects/delete-project/project-deleted.saga.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+import { ICommand, ofType, Saga } from '@nestjs/cqrs';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { ProjectDeleted } from '../events/project-deleted.event';
+import { ScheduleCleanupForProjectUnusedResources } from './schedule-project-unused-resources-cleanup.command';
+
+@Injectable()
+export class ProjectDeletedSaga {
+  @Saga()
+  calculateBlmDefaults = (events$: Observable<any>): Observable<ICommand> =>
```

Comment:
probably a copy/paste leftover?
```suggestion
  deleteProjectResources = (events$: Observable<any>): Observable<ICommand> =>
```